pkg/spoke/submarineragent/config_controller.go

```diff
@@ -573,3 +573,11 @@ func successCondition(gatewayNames []string) metav1.Condition {
 			strings.Join(gatewayNames, ",")),
 	}
 }
+
+func isSpokePrepared(cloudName string) bool {
+	if cloudName == "GCP" || cloudName == "OpenStack" || cloudName == "Azure" {
+		return true
+	}
+
+	return false
+}
```

Comment:
This is duplicated, please create a common method and use it on the hub/spoke.

Reply:
Once we move AWS to the spoke (see https://github.com/stolostron/submariner-addon/issues/218) then this becomes unnecessary so I don't think it's worth creating a common function for this.

Reply:
Okay, but what about the default case? Do we not have to distinguish between the clouds where we have automation vs clouds like VmWare/BareMetal etc which requires manual preparation?